### PR TITLE
Improvement: Upgradle to 7.6.1

### PR DESCRIPTION
**Pull request by robert3005**
## Before this PR
Currently this repo is on 7.5.1 and will not automatically upgarde to 7.6.1

## After this PR

==COMMIT_MSG==
Upgrade gradle to 7.6.1
==COMMIT_MSG==
This is done as a step towards gradle 8 compatibility

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/gradle-consistent-versions](https://github.com/palantir/gradle-consistent-versions), @robert3005! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: build.gradle [hunk 1]
```diff
@@ -98,22 +104,6 @@ gradlePlugin {
...
 }
 
-pluginBundle {
-    website = 'https://github.com/palantir/gradle-consistent-versions'
-    vcsUrl = 'https://github.com/palantir/gradle-consistent-versions'
-    description = 'Compact, constraint-friendly lockfiles for your dependencies'
-    tags = ['versions']
-
-    // Don't publish the other plugins to the plugin portal
-    plugins.removeIf { it.name != 'consistentVersions' }
```
Comment (robert3005) [inline build.gradle:108]: genuinely not sure what is the right substitute of this method

Comment (robert3005) [reply build.gradle:108]: Based https://docs.gradle.org/current/userguide/java_gradle_plugin.html#sec:gradle_plugin_dev_interactions what I have done I think does the same thing as this block

**Comment by robert3005** [comment]
ping, anyone reviewing prs on this repo?

**Review by fawind — APPROVED** [review]
Thanks for fixing!

**Comment by robert3005** [comment]
👍

**Comment by fawind** [comment]
Arg, disqualified. Will get someone to double approve!

**Comment by CRogers** [comment]
:+1:

**Pull request merged**